Log/v1.5_Changes.md

```diff
@@ -0,0 +1,50 @@
+# Version 1.5 Changes
+
+For many many reasons, we're making some significant changes to the program that will go into effect starting June 19, 2017.
+
+This document outlines the planned changes from a high level. More details to follow.
+
+## What's going away
+
+- Stats (i.e. most of what the "game" formerly was)
+- Levels
+- Wandering in the wilderness (lack of learning tracks)
```

Comment:
counter-argument: self-directed learning can still be sequenced and/or have learning tracks, they're just determined by the learner as opposed to the institution. 

if your concern is that someone will read no more 'wandering in the wilderness' as 'no more self-directed learning' then they're reading the right thing. we have removed significant self-direction. learners only have choice once they reach IPC phase, and then their choice is constrained by available snapshot goals and modules.